dpnp/dpnp_iface_histograms.py

```diff
@@ -306,6 +312,61 @@ def digitize(x, bins, right=False):
     return bins.size - dpnp.searchsorted(bins[::-1], x, side=side)
 
 
+def _find_supported_dtype(dt, supported):
```

Comment:
Could you please place the new functions implementation according to a lexicographical order in the file.

Reply:
It doesn't seems that functions are sorted in other modules.
And I'm not sure that's rational behind this?